gnomad_qc/v5/resources/basics.py

```diff
@@ -6,3 +6,26 @@
 dragen_tgp_vds = VariantDatasetResource(
     "gs://gnomad/v5.0/testing/genomes/dragen_tgp_v5.0_test.vds"
 )
+
+
+# TODO: Change default to CURRENT_VERSION.
+def qc_temp_prefix(version: str = "5.0") -> str:
+    """
+    Return path to temporary QC bucket.
+
+    :param version: Version of annotation path to return.
+    :return: Path to bucket with temporary QC data.
+    """
+    return f"gs://gnomad-tmp/gnomad.genomes.v{version}.qc_data/"
+
+
+# TODO: Change default to CURRENT_VERSION.
```

Comment:
why not add CURRENT_VERSION now ? or is that poor practice if there's not a version out

Reply:
Just remembered that CURRENT_VERSION is already defined for v5 in another script, will update